api/src/pcapi/connectors/api_allocine.py

```diff
@@ -77,10 +97,24 @@ def _parse_movie_showtimes(
             raise
 
         data = json_data
-        for err in exc.errors():
-            data = filter_invalid_edge(data, err["loc"])
 
-        return _parse_movie_showtimes(data, try_count + 1)
+        # warning: the reversed sort is very import here. Without it, two (bad)
+        # things could happen:
+        #   1. IndexError because the json data has lost edges and the error
+        #      location index if out of date or
```

Comment:
J'imagine que tu voulais dire ça.

```suggestion
        #      location index is out of range or
```